jme3-core/src/main/resources/Common/MatDefs/Light/PBRLighting.frag

```diff
@@ -1,367 +1,89 @@
 #import "Common/ShaderLib/GLSLCompat.glsllib"
-#import "Common/ShaderLib/PBR.glsllib"
-#import "Common/ShaderLib/Parallax.glsllib"
-#import "Common/ShaderLib/Lighting.glsllib"
 
 varying vec2 texCoord;
+vec2 newTexCoord; 
 #ifdef SEPARATE_TEXCOORD
-  varying vec2 texCoord2;
+    varying vec2 texCoord2;
 #endif
 
-varying vec4 Color;
-
-uniform vec4 g_LightData[NB_LIGHTS];
-uniform vec3 g_CameraPosition;
-uniform vec4 g_AmbientLightColor;
-
-uniform float m_Roughness;
-uniform float m_Metallic;
+#ifdef DISCARD_ALPHA
+    uniform float m_AlphaDiscardThreshold;
+#endif
 
 varying vec3 wPosition;    
 
+varying vec3 wNormal;
+varying vec4 wTangent;
 
-#if NB_PROBES >= 1
-  uniform samplerCube g_PrefEnvMap;
-  uniform vec3 g_ShCoeffs[9];
-  uniform mat4 g_LightProbeData;
-#endif
-#if NB_PROBES >= 2
-  uniform samplerCube g_PrefEnvMap2;
-  uniform vec3 g_ShCoeffs2[9];
-  uniform mat4 g_LightProbeData2;
-#endif
-#if NB_PROBES == 3
-  uniform samplerCube g_PrefEnvMap3;
-  uniform vec3 g_ShCoeffs3[9];
-  uniform mat4 g_LightProbeData3;
-#endif
+#import "Common/ShaderLib/PBRLightingParamsReader.glsllib"
+#import "Common/ShaderLib/PBRLighting.glsllib"
+//Important that these 2 ^ glsllibs are referenced AFTER other variables are declared above. 
```

Comment:
This comment section contains some typos. Could you please replace it with the corrections below?

```java
// It is important that these 2 glsllibs are referenced AFTER the other variables above have been declared. 
// The above variables are declared here (rather than in a glslib) to reduce redundancy, since these variables are likely to be used by more than one glslib.
// Only lighting variables are declared in PBRLighting.glslib, and only basic PBR material params are declared in PBRLightingParamsReader.glslib.
// This allows jme developers to create a fork of this shader and make their own changes before reading the base PBR parameters or before the final lighting calculation.
// For example, you can move texCoords based on g_Time before texReads for a simple moving water/lava effect, or blend values like albedo/roughness after texReads
// but before final lighting calculations to do things like dynamic texture splatting.
```